tide-whisperer.go

```diff
@@ -87,6 +106,9 @@ func main() {
 		log.Fatal(dataAPIPrefix, "Problem loading config: ", err)
 	}
 
+	// Config valid
+	tideWhispererConfig.Set(1)
```

Comment:
This will always be `1` too - if the config is invalid the process will be terminated and metrics will not be scraped.